fix_includes.py

```diff
@@ -1906,7 +1906,8 @@ def _DecoratedMoveSpanLines(iwyu_record, file_lines, move_span_lines, flags):
     # to do the comment-replacing first.
     sort_key = firstline.line
     iwyu_version = iwyu_record.full_include_lines.get(m.group(1), '')
-    if _COMMENT_LINE_RE.search(iwyu_version):  # the iwyu version has comments
+    if _COMMENT_LINE_RE.search(iwyu_version) or (flags.update_comments == 'none'):  
+                                               # the iwyu version has comments
```

Comment:
I'd rephrase this and align it with `sort_key = ...` so you can remove the `# replace the comments` fragments:
```
if ...:
  # there are comments that need to be replaced
  sort_key = iwyu_version
```